src/components/new-request/search.vue

```diff
@@ -363,7 +362,7 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
   readonly EntityType = EntityType
   activeActionGroup = NaN
   showRequestActionTooltip = false
-  business = null as BusinessLookupResultIF
+  business = null
```

Comment:
I'll probably create a type (interface) for this, containing the minimum fields.

Reply:
Done.